agent/pkg/osauth/user.go

```diff
@@ -26,14 +26,14 @@ type User struct {
 func LookupUser(username string) *User {
 	passwdFile, err := os.Open(DefaultPasswdFilename)
 	if err != nil {
-		logrus.Error("Could not open %s", DefaultPasswdFilename)
+		logrus.Errorf("Could not open %s", DefaultPasswdFilename)
```

Comment:
This is changing the code included in the previous commit so we should squash this.

Reply:
Is there any particular reason not to squash the entire PR ?

Reply:
No, please squash it all.